Sources/MeiliSearch/Model/Setting.swift

```diff
@@ -3,53 +3,34 @@ import Foundation
 /**
  Each instance of MeiliSearch has three keys: a master, a private, and a public.
  Each key has a given set of permissions on the API routes.
-*/
+ */
 public struct Setting: Codable, Equatable {
 
     // MARK: Properties
 
-    ///List of ranking rules for a given `Index`.
+    /// List of ranking rules for a given `Index`.
     public let rankingRules: [String]
 
-    ///List of searchable attributes for a given `Index`.
+    /// List of searchable attributes for a given `Index`.
     public let searchableAttributes: [String]
 
-    ///List of displayed attributes for a given `Index`.
+    /// List of displayed attributes for a given `Index`.
     public let displayedAttributes: [String]
 
-    ///List of stop-words for a given `Index`.
+    /// List of stop-words for a given `Index`.
     public let stopWords: [String]
 
-    ///List of synonyms and its values for a given `Index`.
+    /// List of synonyms and its values for a given `Index`.
     public let synonyms: [String: [String]]
 
-    ///Return if a given `Index` allows new fields.
-    public let acceptNewFields: Bool
-
-    public init(
-      rankingRules: [String],
-      searchableAttributes: [String],
-      displayedAttributes: [String],
-      stopWords: [String],
-      synonyms: [String: [String]],
-      acceptNewFields: Bool) {
-      self.rankingRules = rankingRules
-      self.searchableAttributes = searchableAttributes
-      self.displayedAttributes = displayedAttributes
-      self.stopWords = stopWords
-      self.synonyms = synonyms
-      self.acceptNewFields = acceptNewFields
-    }
-
-    ///Tries to decode the JSON object to Setting object.
+    /// Tries to decode the JSON object to Setting object.
     public init(from decoder: Decoder) throws {
         let values = try? decoder.container(keyedBy: CodingKeys.self)
+
```

Comment:
Seems like distinctAttribute is missing here, should a PR be opened to fix this @ppamorim ?